python/semantic_kernel/agents/__init__.py

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+from semantic_kernel.agents.agent_base import AgentBase
+from semantic_kernel.agents.agent_channel import AgentChannel
+from semantic_kernel.agents.chat_completion_agent import ChatCompletionAgent
+from semantic_kernel.agents.chat_history_channel import ChatHistoryChannel
+from semantic_kernel.agents.chat_history_handler import ChatHistoryHandler
+from semantic_kernel.agents.chat_history_kernel_agent import ChatHistoryKernelAgent
+
+__all__ = [
```

Comment:
it would be good to distinguish between base classes and the concrete one, maybe only expose ChatCompletionAgent (and in the future others) here and let folks directly import the others from the relevant files.